models/bridge/zksync/zksync_standard_bridge_flows.sql

```diff
@@ -0,0 +1,168 @@
+{{ config(
+    
+    alias = 'standard_bridge_flows',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["zksync"]\',
+                                "sector",
+                                "bridge",
+                                \'["lgingerich"]\') }}'
+    )
+}}
+
+WITH bridge_events as (
+    SELECT
+         block_time
+        ,CAST(DATE_TRUNC('day', block_time) as date) as block_date
+        ,CAST(DATE_TRUNC('month', block_time) as date) as block_month
+        ,block_number
+        ,tx_hash
+        ,evt_index
+        ,sender_address
+        ,receiver_address
+        ,bridged_token_address
+        ,bridged_token_amount_raw
+        ,source_chain_id
+        ,destination_chain_id
+        ,source_chain_name
+        ,destination_chain_name
+    FROM (
+        -- Deposit ETH from Ethereum to zkSync Era
+        SELECT
+             npr.evt_block_time as block_time
+            ,npr.evt_block_number as block_number
+            ,npr.evt_tx_hash as tx_hash
+            ,npr.evt_index
+            ,et."from" as sender_address
+            ,COALESCE(d.to, zt.to) as receiver_address -- d.to is null if there is no matching ERC20 deposit tx hash. This is used to handle the log of an 'if statement'.
+            ,0x0000000000000000000000000000000000000000 as bridged_token_address
+            ,CAST(JSON_EXTRACT_SCALAR(npr.transaction, '$.reserved[0]') as UINT256) as bridged_token_amount_raw
+            ,UINT256 '1' as source_chain_id
+            ,UINT256 '324' as destination_chain_id
+            ,'Ethereum' as source_chain_name
+            ,'zkSync Era' as destination_chain_name
+        FROM {{ source('zksync_v2_ethereum', 'DiamondProxy_evt_NewPriorityRequest') }} npr
+        LEFT JOIN {{ source('ethereum', 'transactions') }} et ON npr.evt_tx_hash = et.hash
+        LEFT JOIN {{ source('zksync', 'transactions') }} zt ON npr.txHash = zt.hash
+        LEFT JOIN {{ source('zksync_v2_ethereum', 'L1ERC20Bridge_evt_DepositInitiated') }} d ON npr.evt_tx_hash = d.evt_tx_hash AND npr.evt_index = d.evt_index
+        {% if is_incremental() %}
+        WHERE {{ incremental_predicate('npr.evt_block_time') }}
+        {% endif %}
+
+        UNION ALL
+
+        -- Deposit ERC-20 from Ethereum to zkSync Era
+        SELECT
+             d.evt_block_time as block_time
+            ,d.evt_block_number as block_number
+            ,d.evt_tx_hash as tx_hash
+            ,d.evt_index
+            ,d."from" as sender_address
+            ,d.to as receiver_address
+            ,d.l1Token as bridged_token_address
+            ,d.amount as bridged_token_amount_raw
+            ,UINT256 '1' as source_chain_id
+            ,UINT256 '324' as destination_chain_id
+            ,'Ethereum' as source_chain_name
+            ,'zkSync Era' as destination_chain_name
```

Comment:
not a requirement if you have downstream use cases, but typically we lowercase blockchain names